python/hyperon/exts/snet_io/snet_gate.py

```diff
@@ -100,7 +101,62 @@ def __call__(self, command_a, *args_a):
             service_client = self.create_service_client(*args, **kwargs)
             return [service_client.get_operation_atom()]
         return [E(S('Error'), E(S('snet-sdk'), command_a, *args_a),
-                    ValueAtom(f'unknown command {repr(command_a)}'))]
+                  ValueAtom(f'unknown command {repr(command_a)}'))]
+
+len_threshold = 50
+current_len = 0
```

Comment:
What will happen when the function is called twice? Will `current_len` behave correctly?

Reply:
Well, it should be set to 0 (line 158 sets it to 0 after every subatom was processed). I'll check just to be sure.

Reply:
@Necr0x0Der  I've checked (though I needed to do some changes to init_sdk since snet-sdk was updated and now its init is slightly different, probably i need to make new pr for those changes?) and current_len is 0 on the start of pretty print. And it can be seen on line 154.

Reply:
OK, but still... Are they used only in `pretty_print_atoms` and its subfunctions? In this case, it's better to not declare them as `global`, but rather introduce them in `pretty_print_atoms` and use `nonlocal` for them in subfunctions

Reply:
I'll try this out. I haven't been aware of `nonlocal`. Of course it would be better to introduce them inside function I just wasn't been able to make them work correctly. If `nonlocal `will help, I'll push changes tomorrow.

Reply:
@Necr0x0Der  Okay, it works. Thank you. I've pushed new commit with changes regarding current_len and threshold